File: 0025-vowel-repeater/vowel-repeater.js
function repeatVowels(str) {
    const vowels = "aeiouAEIOU";
    let count = 0;
    let result = "";
    
    for (let i = 0; i < str.length; i++) {
        const ch = str[i];
        
        if (vowels.includes(ch)) {
            count++;
            if (count === 1) {
                result += ch;
            } else {
                result += ch + ch.toLowerCase().repeat(count - 1);
            }
        } else {
            result += ch;
        }
    }
    
    return result;
}

File: 0025-vowel-repeater/vowel_repeater.py
def repeat_vowels(s):
    vowels = "aeiouAEIOU"
    count = 0
    result = ""
    
    for ch in s:
        if ch in vowels:
            count += 1
            if count == 1:
                result += ch
            else:
                result += ch + ch.lower() * (count - 1)
        else:
            result += ch
    
    return result